src/group/managed_group/mod.rs

```diff
@@ -92,17 +94,15 @@ impl ManagedGroup {
     pub fn add_members(
         &mut self,
         credential_bundle: &CredentialBundle,
-        aad: &[u8],
```

Comment:
Why is aad removed here, as well as in `remove_members`, but not in `propose_add_members`? I think `aad` is something that applications might want to use, so why not expose it?

Reply:
I forgot to remove it there as well.
AAD is important but will probably not be used often. It can be set with `.set_aad()` instead, so that the API is cleaner.

Reply:
Wouldn't it be used to communicate with the DS? I could imagine that deployers will want some way of communicating with the DS and aad could be it. Also, `set_aad()` hints that aad is something that has to be set on a per-group level, rather than on a per-message level, which is a bit misleading.

Reply:
That's precisely the design question. Since I'm assuming that AAD will be rarely used, in particular on a per-message basis, I'm stripping it from the functions that will be used often.
If implementation still want to use it, they can change the group's AAD with `.set_aad()`.
If we see that the AAD will be used somewhere we can always revert this decision, but right now we don't have any concrete example of where it would be useful.